site/envoy-versions-schema.json

```diff
@@ -1,96 +0,0 @@
-{
```

Comment:
is this going to be a breaking change?

Reply:
we don't have anything except docs referring to the location of this file. but I will rename it back for this PR anyway!

Reply:
ps to answer no.. it isn't a breaking change, but before our json schema had a bug in it, so there's that also :D